README.md

```diff
@@ -168,6 +168,12 @@ Note: Don't use `@` symbol when adding username.
 When you are done with the setup, you should host your website online.
 We highly recommend to read through the [Deploying on Github Pages](https://create-react-app.dev/docs/deployment/#github-pages) docs for React.
 
+#### Configuring GitHub Actions
+
+- [Create a second GitHub Personal Access Token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) for use by the [JamesIves GitHub Pages Deploy GitHub Actions](https://github.com/JamesIves/github-pages-deploy-action) with `public_repo, read:org, workflow` checked
+  - On your repository fork, create a Secret called `ACCESS_TOKEN` under the <b>Settings/Secrets</b> tab
```

Comment:
Change the middle of the sentence to
'... create a [repository secret](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#creating-encrypted-secrets-for-a-repository) with the name `ACCESS_TOKEN`...'

Reply:
Remove the above lines.